### PR TITLE
add useMap hooks

### PR DESCRIPTION
**Pull request by luchx**

### CODE REVIEW

**Review by brickspert — APPROVED** [review]
LGTM

File: packages/hooks/src/useSet/demo/demo1.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /**
  * title: Default usage
- * desc: Pass in a set acceptable parameter.
+ * desc: Pass in a Set acceptable parameter.
```
Comment (ttys026) [inline packages/hooks/src/useSet/demo/demo1.tsx:3]: Set -> Map

File: packages/hooks/src/useSet/demo/demo1.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
  * 
  * title.zh-CN: 默认用法
  * desc.zh-CN: 传入一个 Set 可接受的参数。
```
Comment (ttys026) [inline packages/hooks/src/useSet/demo/demo1.tsx:6]: Set -> Map

File: packages/hooks/src/useSet/demo/demo1.tsx (hunk 1 above)
Comment (brickspert) [reply packages/hooks/src/useSet/demo/demo1.tsx:3]: 尴尬了~~

**Pull request merged**